Sources/SpeziOnboarding/OnboardingView.swift

```diff
@@ -86,7 +117,6 @@ public struct OnboardingView<TitleView: View, ContentView: View, ActionView: Vie
     
     /// Creates the default style of the `OnboardingView` uses a combination of an ``OnboardingTitleView``, ``OnboardingInformationView``,
     /// and ``OnboardingActionsView``.
-    ///
```

Comment:
@akanshyabhat One more comment: Why did you remove this line?

Reply:
Sorry, I can put it back in. I think I got confused by a comment that Paul left earlier but I shouldn't have removed it.